tests/test_frontdoor.py

```diff
@@ -682,31 +663,7 @@ def test_feedback_with_session():
     assert page.has_one("h1:contains('Do you own the property?')")
 
     page = page.click(contains="feedback")
-    form = page.get_form()
-    form["satisfaction"] = "Somewhat satisfied"
-    form["usage-reason"] = "To find ways to reduce my energy bills"
-    form["guidance"] = "Agree"
-    form["accuracy"] = "Neutral"
-    form["advice"] = "Agree"
-    form["more-detail"] = "Improvement comment"
-
-    page = form.submit().follow()
-
-    assert page.has_one("h1:contains('Thank you for your feedback')")
-
-    feedback = frontdoor_models.Feedback.objects.latest("created_at")
-    assert feedback.data["satisfaction"] == "Somewhat satisfied"
-    assert feedback.data["usage-reason"] == "To find ways to reduce my energy bills"
-    assert feedback.data["guidance"] == "Agree"
-    assert feedback.data["accuracy"] == "Neutral"
-    assert feedback.data["advice"] == "Agree"
-    assert feedback.data["more-detail"] == "Improvement comment"
-
-    feedback_session_id = page.path.split("/")[3]
-    assert uuid.UUID(feedback_session_id)
-
-    page = page.click(contains="Return to your application")
```

Comment:
There doesn't seem to be a way to go back to previous page in this case as it doesn't maintain a history of visited page?

If it is the case then we might just delete these tests?

Reply:
Yep, since we don't look after this page we can't add one either, purge the test :D